source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRViewPrivate.h

```diff
@@ -77,4 +77,7 @@ typedef void (^ObserverActionBlockForBaseAction)(NSObject<ACOIResourceResolver>
 - (ACOInputResults *)dispatchAndValidateInput:(ACRColumnView *)parent;
 
 - (void)removeObserver:(NSObject *)observer forKeyPath:(NSString *)path onObject:(NSObject *)object;
+
+- (void)setContext:(ACORenderContext *)context;
```

Comment:
>(void)setContext:(ACORenderContext *)context; [](http://example.com/codeflow?start=2&length=45)

what is the purpose of this Context setter? From the mock sample I get the usage could cover scenarios as knowing all actions in an action set having icons but I don't see the actual method being enforced in any test

Reply:
context is proving the value for icon placement. the mock value will allow choosing the execution branches.